src/leap/mail/incoming/service.py

```diff
@@ -729,7 +729,7 @@ def signal_deleted(doc_id):
             d.addCallback(signal_deleted)
             return d
 
-        d = self._inbox.add_raw_message(data, (self.RECENT_FLAG,))
+        d = self._inbox.add_msg(data, (self.RECENT_FLAG,))
```

Comment:
`add_msg` now does not set the date as `add_raw_message` was doing. We may care more about the date that comes in the message header, but then in `add_msg` we are setting the date as "" removing the date from the header.